src/api/services/languageService.ts

```diff
@@ -0,0 +1,139 @@
+    [Language.Italian]: { name: 'Italian', iconSuffix: 'italy' },
+    [Language.Japanese]: { name: 'Japanese', iconSuffix: 'japan' },
+    [Language.Korean]: { name: 'Korean', iconSuffix: 'south-korea' },
+    [Language.Vietnamese]: { name: 'Vietnamese', iconSuffix: 'vietnam' },
+    [Language.Chinese]: { name: 'Chinese', iconSuffix: 'china' },
+    [Language.Spanish]: { name: 'Spanish', iconSuffix: 'spain' },
+    [Language.Turkish]: { name: 'Turkish', iconSuffix: 'turkey' },
+    [Language.German]: { name: 'German', iconSuffix: 'germany' },
+    [Language.PortugueseBrazil]: { name: 'Portuguese (Brazil)', iconSuffix: 'brazil' },
+    [Language.Polish]: { name: 'Polish', iconSuffix: 'poland' },
+    [Language.Hungarian]: { name: 'Hungarian', iconSuffix: 'hungary' },
+    [Language.Czech]: { name: 'Czech', iconSuffix: 'czechia' },
+    [Language.Ukrainian]: { name: 'Ukrainian', iconSuffix: 'ukraine' },
+    [Language.Catalan]: { name: 'Catalan' },
+    [Language.Arabic]: { name: 'Arabic' },
+    [Language.Bulgarian]: { name: 'Bulgarian', iconSuffix: 'bulgaria' },
+    [Language.Swedish]: { name: 'Swedish', iconSuffix: 'sweden' },
+    [Language.Finnish]: { name: 'Finnish', iconSuffix: 'finland' },
+    [Language.Romanian]: { name: 'Romanian', iconSuffix: 'romania' },
+    [Language.Tagalog]: { name: 'Tagalog', iconSuffix: 'philippines' },
+    [Language.Macedonian]: { name: 'Macedonian', iconSuffix: 'north-macedonia' },
+    [Language.Slovene]: { name: 'Slovene', iconSuffix: 'slovenia' },
+    [Language.Hebrew]: { name: 'Hebrew', iconSuffix: 'israel' },
+    [Language.Indonesian]: { name: 'Indonesian', iconSuffix: 'indonesia' },
+    [Language.Danish]: { name: 'Danish', iconSuffix: 'denmark' },
+    [Language.Latvian]: { name: 'Latvian', iconSuffix: 'latvia' },
+    [Language.Lithuanian]: { name: 'Lithuanian', iconSuffix: 'lithuania' },
+    [Language.Dutch]: { name: 'Dutch', iconSuffix: 'netherlands' },
+    [Language.Esperanto]: { name: 'Esperanto' },
+    [Language.Greek]: { name: 'Greek', iconSuffix: 'greece' },
+    [Language.Hindi]: { name: 'Hindi', iconSuffix: 'india' },
+    [Language.Thai]: { name: 'Thai', iconSuffix: 'thailand' },
+    [Language.Urdu]: { name: 'Urdu', iconSuffix: 'pakistan' },
+    [Language.Persian]: { name: 'Persian' },
+    [Language.Irish]: { name: 'Irish', iconSuffix: 'ireland' },
+    [Language.Latin]: { name: 'Latin' },
+    [Language.Portuguese]: { name: 'Portuguese (Portugal)', iconSuffix: 'portugal' },
+    [Language.Malay]: { name: 'Malay', iconSuffix: 'malaysia' },
+    [Language.Norwegian]: { name: 'Norwegian', iconSuffix: 'norway' },
+    [Language.Slovak]: { name: 'Slovak', iconSuffix: 'slovakia' },
+    [Language.ScottishGaelic]: { name: 'Scottish Gaelic' },
+  };
+
+  /**
+   * Converts a certain language to readable equivalent.
+   * @param value Language.
+   */
+  export const toReadable = (value: Language): string => MAP_LANGUAGE_INFO[value]?.name ?? '';
```

Comment:
Isn't it impossible to for `MAP_LANGUAGE_INFO[value]?.name` to be nullish?
So I think optional chaining and nullish operator are unnecessary.